tests/acceptance/features/apiOcm/createInvitation.feature

```diff
@@ -155,36 +97,47 @@ Feature: create invitation
   Scenario: user cannot see expired invitation tokens
     Given using server "LOCAL"
     And the config "OCM_OCM_INVITE_MANAGER_TOKEN_EXPIRATION" has been set to "1s"
-    When "Alice" creates the federation share invitation
+    And "Alice" has created the federation share invitation
+    When the user waits "2" seconds for the invitation token to expire
+    And "Alice" lists the created invitations
     Then the HTTP status code should be "200"
     And the JSON data of the response should match
       """
       {
-        "type": "object",
-        "required": [
-          "expiration",
-          "token"
-        ],
-        "properties": {
-          "expiration": {
-            "type": "integer",
-            "pattern": "^[0-9]{10}$"
-          },
-          "token": {
-            "type": "string",
-            "pattern": "%fed_invitation_token%"
-          }
-        }
+        "type": "array",
+        "minItems": 0,
+        "maxItems": 0
       }
       """
-    And the user waits "2" seconds for the token to expire
+
+
+  Scenario: user lists created invitation
```

Comment:
similar scenario for deleted step in `Scenario: user creates invitation with valid email and description`